### PR TITLE
Rubocop is your friend part 3

### PR DESCRIPTION
**Pull request by DavidKang**
@openSUSE/open-build-service Please review and let us know if there is any cop you don't agree on.

The examples of the cops we apply here can be found here:

* http://rubocop.readthedocs.io/en/latest/cops_style/#stylezerolengthpredicate
* http://rubocop.readthedocs.io/en/latest/cops_style/#styleyodacondition
* http://rubocop.readthedocs.io/en/latest/cops_style/#stylestringliteralsininterpolation

### CODE REVIEW

**Comment by DavidKang** [comment]
[(5) P6: Fix rubocop process](https://trello.com/c/JAeV3sf1/631-5-p6-fix-rubocop-process)

File: src/api/app/views/attribute/namespace_definition.xml.builder [hunk 1]
```diff
@@ -1,5 +1,5 @@
 abies = @an.attrib_namespace_modifiable_bies.includes([:user, :group])
-if abies.length > 0
+if !abies.empty?
```
[the remark this answers is not in the document]
Comment (DavidKang) [reply src/api/app/views/attribute/namespace_definition.xml.builder]: @ChrisBr, because it is an `if....else` statement.

**Comment by bgeuken** [comment]
@mschnitzer Please review, and merge if the changes are fine with you

**Comment by mschnitzer** [comment]
what about the dangling `codecov` ?

**Comment by Ana06** [comment]
@mschnitzer 

> what about the dangling codecov ?

This is not the only PR were it is like that... I guess we can merge the PR as style changes shouldn't  affect coverage. Although we should take a look at what the problem with codecov is.

**Pull request merged**